jans-cedarling/bindings/cedarling_python/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cedarling_python"
+version = "0.1.0"
```

Comment:
again the version needs to match the starting version of jans versions . In this case 1.1.6

Reply:
Thanks, fixed.
Do we also need to change the version for the sparkv package? This code was forked.